cmd/mount.go

```diff
@@ -321,6 +322,10 @@ var mountCmd = &cobra.Command{
 			options.Components = append(options.Components[:1], append([]string{"entry_cache"}, options.Components[1:]...)...)
 		}
 
+		if config.IsSet("xload") {
+			config.Set("read-only", "true") // preload is only supported in read-only mode
+		}
```

Comment:
Incorrect, even if xload is not present in the components section, but is present in the config file this block will execute making the mount read-only.